Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -23435,6 +23435,126 @@
+        <Food value="-1"/>
+        <AbilArray Link="stop"/>
+        <AbilArray Link="attack"/>
+        <AbilArray Link="move"/>
+        <AbilArray Link="AP_BurrowPygaliskDown"/>
+        <BehaviorArray Link="OnCreep"/>
+        <BehaviorArray Link="AP_MalignantCreepFix"/>
+        <WeaponArray Link="AP_Pygalisk"/>
+        <WeaponArray Link="AP_PygaliskHeadAttack"/>
+        <AbilArray Link="AP_StimpackPygalisk"/>
+        <CardLayouts>
+            <LayoutButtons Face="Move" Type="AbilCmd" AbilCmd="move,Move" Row="0" Column="0"/>
+            <LayoutButtons Face="Stop" Type="AbilCmd" AbilCmd="stop,Stop" Row="0" Column="1"/>
+            <LayoutButtons Face="MoveHoldPosition" Type="AbilCmd" AbilCmd="move,HoldPos" Row="0" Column="2"/>
+            <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
+            <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
+            <LayoutButtons Face="BurrowDown" Type="AbilCmd" AbilCmd="AP_BurrowPygaliskDown,Execute" Row="2" Column="4"/>
+            <LayoutButtons Face="AP_Stim" Type="AbilCmd" AbilCmd="AP_StimpackPygalisk,Execute" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_PygaliskDucalBlades" Type="Passive" Requirements="AP_HavePygaliskDucalBlades" Row="1" Column="0"/>
+            <LayoutButtons Face="AP_PygaliskCombatCarapace" Type="Passive" Requirements="AP_HavePygaliskCombatCarapace" Row="1" Column="1"/>
+        </CardLayouts>
+        <EditorCategories value="ObjectType:Unit,ObjectFamily:Campaign"/>
+        <Fidget>
+            <ChanceArray index="Anim" value="50"/>
+            <ChanceArray index="Idle" value="50"/>
+        </Fidget>
+        <GlossaryPriority value="181"/>
+        <TacticalAI value="BabyUltralisk"/>
+    </CUnit>
+    <CUnit id="AP_PygaliskBurrowed">
+        <Name value="Unit/Name/AP_Pygalisk"/>
+        <Description value="Button/Tooltip/AP_Pygalisk"/>
+        <DeathRevealRadius value="3"/>
+        <Race value="Zerg"/>
+        <Mob value="Campaign"/>
+        <FlagArray index="PreventDestroy" value="1"/>
+        <FlagArray index="UseLineOfSight" value="1"/>
+        <FlagArray index="Cloaked" value="1"/>
+        <FlagArray index="Buried" value="1"/>
+        <FlagArray index="AIThreatGround" value="1"/>
+        <FlagArray index="ArmySelect" value="1"/>
+        <PlaneArray index="Ground" value="1"/>
+        <Collide index="Burrow" value="1"/>
+        <Attributes index="Biological" value="1"/>
+        <LifeStart value="75"/>
+        <LifeMax value="75"/>
+        <LifeArmor value="3"/>
+        <LifeRegenRate value="0.2734"/>
+        <LifeArmorName value="Unit/LifeArmorName/ZergGroundArmor"/>
+        <ShieldArmorName value="Unit/ShieldArmorName/ZergWildMutation"/>
+        <Mover value="Burrowed"/>
+        <Sight value="6"/>
+        <Food value="-1"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="50"/>

```

Comment:
![Snímek obrazovky pořízený 2024-11-28 17-46-10](https://github.com/user-attachments/assets/1c73e395-7b03-461d-a17b-6ee602e32cda)
Should match the unburrowed form